juju-tutorials/how-to-fixing-attach-volume-to-a-k8s-instance.md

```diff
@@ -0,0 +1,336 @@
+[comment]: <> (How To Perform Fixing Attach Volume to Instance)
+
+## pre-install
+
+Duration: 120:00
+
+This description of how the deployment of `charmed kubernetes` is performed on a OpenStack server, where OpenStack is deployed to another OpenStack. (openstack-on-openstack).
```

Comment:
Suggestion: This tutorial has been tested on a CDK (link to the CDK bundle) environment deployed on an OpenStack (link to openstack base) environment. The OpenStack environment was deployed on top of another managed OpenStack service.